devops/mac.yml

```diff
@@ -0,0 +1,66 @@
+trigger: none
+pr: none
+
+schedules:
+  - cron: "30 5 * * *" # Time is UTC
+    displayName: Nightly Code Coverage Build
+    branches:
+      include:
+        - master
+    always: true
+
+strategy:
+  matrix:
+    linux:
+      imageName: "ubuntu-16.04"
+    mac:
+      imageName: "macos-10.13"
+    windows:
+      imageName: "vs2017-win2016"
+
+pool:
+  vmImage: $(imageName)
+
+steps:
+  - task: UsePythonVersion@0
+    displayName: "Use Python 3.6"
+    inputs:
+      versionSpec: 3.6
+
+  - script: pip3 install torch===1.3.0 torchvision===0.4.1 -f https://download.pytorch.org/whl/torch_stable.html --user
+    displayName: Install pytorch packages
+
+  - script: pip install --upgrade pip --user
+    displayName: Upgrade pip to latest
+
+  - script: pip install -r requirements.txt
+    displayName: Install required pip packages
+
+  - script: pip install -e ./python
+    displayName: Install interpret-extensions
```

Comment:
interpret-community

Reply:
I deleted mac.yml as it will use Walter's mac